NOTICE

```diff
@@ -7,9 +7,9 @@ listed below.  Your use of this material within the component is also subject to
 conditions of these licenses.  The LICENSE file contains the text of all the licenses which apply
 within this component.
 
-The src/cunit/cunit_lgpl directory tree within this component contains some software which is
-Licensed under the LGPL-2.0 License.  The src/cunit/cunt_lgpl directory contains its own license file,
-called COPYING, which describes the licensing of the software under the src/cunit/cunit_lgpl directory.
+The src/c_source/cunit_lgpl directory tree within this component contains some software which is
```

Comment:
will you need cpp_source/gtest_lgpl?

Reply:
I don't think so, as we are not modifying anything from gtest. Just building and using it.